src/main/java/hw_4/Task4.java

```diff
@@ -0,0 +1,54 @@
+package hw_4;
+
+import java.util.Scanner;
+
+public class Task4 {
+    public static void main(String[] args) {
+
+        String[] arrCity = {
+                "Київ", "Харків", "Львів", "Одеса", "Дніпро", "Запоріжжя", "Вінниця", "Полтава", "Чернігів", "Херсон",
+                "Амстердам", "Варшава", "Лондон", "Берлін", "Париж", "Мадрид", "Рим", "Прага", "Будапешт", "Відень",
+                "Токіо", "Сеул", "Пекін", "Сінгапур", "Гонконг", "Шанхай", "Тайбей", "Бангкок", "Куала-Лумпур", "Делі",
+                "Москва", "Санкт-Петербург", "Мінськ", "Алмати", "Ташкент", "Баку", "Єреван", "Тбілісі", "Ашхабад", "Бішкек",
+                "Нью-Йорк", "Лос-Анджелес", "Чикаго", "Х'юстон", "Фінікс", "Філадельфія", "Сан-Антоніо", "Сан-Дієго", "Даллас", "Сан-Хосе",
+                "Мехіко", "Гвадалахара", "Монтеррей", "Пуебла", "Толука", "Тіхуана", "Леон", "Сьюдад-Хуарес", "Торреон", "Меріда",
+                "Буенос-Айрес", "Ріо-де-Жанейро", "Сан-Паулу", "Сантьяго", "Ліма", "Богота", "Каракас", "Кіто", "Ла-Пас", "Монтевідео",
+                "Кейптаун", "Йоганнесбург", "Каїр", "Лагос", "Найробі", "Аддис-Абеба", "Хартум", "Кампала", "Дар-ес-Салам", "Алжир",
+                "Сідней", "Мельбурн", "Брисбен", "Перт", "Аделаїда", "Канберра", "Окленд", "Веллінгтон", "Крайстчерч", "Гамільтон",
+                "Торонто", "Монреаль", "Ванкувер", "Калгарі", "Оттава", "Едмонтон", "Вінніпег", "Гамільтон", "Квебек", "Саскатун"
+        };
+
+        boolean fCity = false;
+        boolean fQuit = false;
+
+        System.out.println("Давай пограємо у «Міста». Ти перший називай!");
+
+        Scanner scanner = new Scanner(System.in);
+
+        do {
+            String userInput = scanner.nextLine();
+            char firstLetter = userInput.toLowerCase().charAt(userInput.length() - 1);
```

Comment:
firstLetterTiFindInArray or something like that?

Reply:
fixed